### PR TITLE
[sdk-1.5.0-hotfix] Update CHANGELOG for release

### PR DESCRIPTION
**Pull request by CodeBlanch**
**Note: This PR is targeting main-1.5.0 branch.**

## Changes

* Update CHANGELOG for 1.5.1 release.

## Merge requirement checklist

* [X] [CONTRIBUTING](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/CONTRIBUTING.md) guidelines followed (nullable enabled, static analysis, etc.)
* [X] Appropriate `CHANGELOG.md` files updated for non-trivial changes

### CODE REVIEW

**Review by alanwest — APPROVED** [review]

**Review by utpilla — APPROVED** [review]

File: src/OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.5.1
```
Comment (utpilla) [inline src/OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs/CHANGELOG.md:5]: We don't have to release this package.

Comment (utpilla) [reply src/OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs/CHANGELOG.md:5]: Same for both the Prometheus exporter packages.

Comment (CodeBlanch) [reply src/OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs/CHANGELOG.md]: Updated

File: src/OpenTelemetry.Exporter.Prometheus.AspNetCore/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.5.1
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.Prometheus.AspNetCore/CHANGELOG.md]: Keep prerelease

**Review by reyang — APPROVED** [review]

**Pull request merged**